students/DrewSmith/session02/print_grid.py

```diff
@@ -0,0 +1,34 @@
+def print_simple_grid():
+    divide = "+ " + ("- " * 4) + "+ " + ("- " * 4) + "+"
+    length = "| " + ("  " * 4) + "| " + ("  " * 4) + "|"
+    print(divide)
+    for __ in range(3):
+        for __ in range(4):
+            print(length)
+        print(divide)
+
+def print_grid(n):
+    square_width = ""
+    for index in range(n):
+        square_width += " " if index % 2 == 0 else "-"
```

Comment:
interesting! can you figure a way to do it without the loop -- maybe some multipliation?

also -- does it work right for both even and odd n?